apps/web/src/components/NftSelector/NftSelectorTokens.tsx

```diff
@@ -0,0 +1,118 @@
+import { useCallback, useMemo } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+
+import { CREATOR_BETA_ANNOUNCEMENT_SEEN } from '~/constants/storageKeys';
+import { NftSelectorTokensFragment$key } from '~/generated/NftSelectorTokensFragment.graphql';
+import usePersistedState from '~/hooks/usePersistedState';
+import { contexts } from '~/shared/analytics/constants';
+import { GalleryElementTrackingProps } from '~/shared/contexts/AnalyticsContext';
+import { Chain } from '~/shared/utils/chains';
+
+import { Button } from '../core/Button/Button';
+import GalleryLink from '../core/GalleryLink/GalleryLink';
+import { VStack } from '../core/Spacer/Stack';
+import { BaseM, TitleCondensed } from '../core/Text/Text';
+import { NftSelectorContractType } from './NftSelector';
+import { NftSelectorLoadingView } from './NftSelectorLoadingView';
+import { NftSelectorView } from './NftSelectorView';
+type Props = {
+  selectedFilter: string;
+  isLocked: boolean;
+  tokenRefs: NftSelectorTokensFragment$key;
+  selectedContractAddress: string | null;
+  onSelectContract: (contract: NftSelectorContractType) => void;
+  onSelectToken: (tokenId: string) => void;
+  eventFlow: GalleryElementTrackingProps['eventFlow'];
+  selectedNetworkView: Chain;
+  hasSearchKeyword: boolean;
+  handleRefresh: () => void;
+};
+
+export default function NftSelectorTokens({
+  selectedFilter,
+  isLocked,
+  selectedContractAddress,
+  onSelectContract,
+  onSelectToken,
+  tokenRefs,
+  selectedNetworkView,
+  hasSearchKeyword,
+  handleRefresh,
+  eventFlow,
+}: Props) {
+  const tokens = useFragment(
+    graphql`
+      fragment NftSelectorTokensFragment on Token @relay(plural: true) {
+        ...NftSelectorViewFragment
+      }
+    `,
+    tokenRefs
+  );
+
+  const [creatorBetaAnnouncementSeen, setCreatorBetaAnnouncementSeen] = usePersistedState(
+    CREATOR_BETA_ANNOUNCEMENT_SEEN,
+    'false'
+  );
+
+  const handleContinueCreatorBetaClick = useCallback(() => {
+    setCreatorBetaAnnouncementSeen('true');
```

Comment:
nit: you can just use raw `false` and `true` values without wrapping it in a string; `usePersistedState` will handle stringification

Reply:
non-blocker: would prefer to do this via backend feature flag, unless we won't have announcement up for a while